lib/javascripts/base_app.js

```diff
@@ -226,6 +226,12 @@ BaseApp.extend = function(appPrototype) {
 
   App.prototype = _.extend({}, BaseApp.prototype, appPrototype);
 
+  Object.defineProperty(App.prototype, "settings", {
+    get() {
+      return this._metadata.settings;
```

Comment:
The intention of doing this after `_.extend` is not very clear in the code itself, maybe add a comment? Or can we do something like this instead ``return this._metadata && this._metadata.settings;``?

Reply:
Hello